addons/point_of_sale/static/src/js/models.js

```diff
@@ -1482,6 +1483,9 @@ exports.Orderline = Backbone.Model.extend({
                 return t.id === el;
             }));
         });
+        product_taxes = _.map(product_taxes, function(tax) {
+            return self._map_tax_fiscal_position(tax);
+        });
```

Comment:
Small style thing, I would personally prefer:

```javascript
product_taxes = _.map(product_taxes, this._map_tax_fiscal_position.bind(this));
```

Then you don't need the ```self``` anymore either. It does the same thing of course.